AutomaticInterface/AutomaticInterface/Builder.cs

```diff
@@ -13,9 +13,11 @@ public static class Builder
 
     private static readonly SymbolDisplayFormat MethodSignatureDisplayFormat =
         new(
+            genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
             memberOptions: SymbolDisplayMemberOptions.IncludeParameters,
             parameterOptions: SymbolDisplayParameterOptions.IncludeType
-                | SymbolDisplayParameterOptions.IncludeParamsRefOut
+                | SymbolDisplayParameterOptions.IncludeParamsRefOut,
+            typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces
```

Comment:
I have added this to prevent types with the same name from being seen as identical. I can't write a test for this case yet, as the generator can't currently handle types with the same name anyway (see #55)